src/lib.rs

```diff
@@ -71,17 +71,17 @@ use std::io::{ErrorKind::NotFound, Write};
 use std::path::{Path, PathBuf};
 use thiserror::Error;
 
-#[cfg(not(any(feature = "toml_conf", feature = "yaml_conf")))]
+#[cfg(not(any(feature = "toml_conf", feature = "yaml_conf", feature = "ron_conf")))]
 compile_error!(
     "Exactly one config language feature must be enabled to use \
-confy.  Please enable one of either the `toml_conf` or `yaml_conf` \
-features."
+confy.  Please enable one of either the `toml_conf`, `yaml_conf` \
```

Comment:
```suggestion
confy.  Please enable one of either the `toml_conf`, `yaml_conf`, \
```

I like having the extra comma there for clarity that the last two are not an exclusive set (known as the [oxford comma](https://www.grammarbook.com/blog/commas/oxford-comma/))

Note: this suggestion may fail `rustfmt`

Reply:
Resolved in https://github.com/rust-cli/confy/pull/71/commits/a4716bbb659181507f9a957d40b052993edcb7ad